readyplayerme/rpm-unity-sdk-avatar-creator
# [SDK-10] Responsive UI

## Description

- UI panels and prefabs are updated to correctly align in wide and narrow screens.

## Changes

#### Updated
- AvatarCreatorExample Sample project updated.
- AvatarSelection, FullbodyButton, and DefaultAvatarButton prefabs are updated.



## How to Test

- Import RPM SDK core as the first step
- Import Custom Avatar Creator package via https://github.com/readyplayerme/rpm-unity-sdk-avatar-creator.git#feature/ui-updates
- Import the Avatar Creator Sample project
- Set the screen to Free Aspect from Game Window
- Stretch the game window into narrow and wide sizes in every editor screen to observe clamping UI elements.

## Checklist
